tests/parameters/test_base.py

```diff
@@ -12,24 +12,24 @@
 from .util import check_sampling, check_value_numvalue_conversion
 
 
-class DummyParameter(BaseParameter):
-    def check_value(self):
+class PseudoParameter(BaseParameter):
+    def check_value(self, value):
         pass
 
 
 def test_base_parameter():
     bounds = np.array((-4, 8))
     prior = Uniform()
 
-    p = DummyParameter(
+    p = PseudoParameter(
         "foo",
         prior,
         transformation=ZeroOneFloat(bounds),
         is_continuous=False,
         is_ordered=False,
         num_values=np.inf,
     )
-    q = DummyParameter(
+    q = PseudoParameter(
```

Comment:
I feel like there is a learning experience hidden in this change. Would you care to reveal it? :relaxed:

Reply:
It is considered to be an not-inclusive word. I usually take a look at the  [AirBnB's list](https://github.com/airbnb/inclusion/blob/main/infractions.yml#L25-L32) to find alternatives, but in this specific case, I felt `Pseudo` fits a bit better than e.g. `Stub`.